support-frontend/assets/helpers/supporterPlus/benefitsThreshold.ts

```diff
@@ -2,12 +2,20 @@ import type {
 	ContributionType,
 	RegularContributionType,
 } from 'helpers/contributions';
+import type { CountryGroup } from 'helpers/internationalisation';
 import type { IsoCurrency } from 'helpers/internationalisation/currency';
 import { productCatalog } from 'helpers/productCatalog';
 import { getContributionType } from 'helpers/redux/checkout/product/selectors/productType';
 import type { ContributionsState } from 'helpers/redux/contributionsStore';
 import { isRecurring } from './isContributionRecurring';
 
+export type ProductsThresholdDefined = 'SupporterPlus' | 'TierThree';
```

Comment:
this feels like another place where we'd have to add products' types - when ideally it could come from the product catalog. See this PR that added a type (still todo to restrict it). Not sure what the solution is but maybe there's another way to encode this threshold / not threshold info
https://github.com/guardian/support-frontend/pull/6133

Reply:
Was looking for that `productId` type, even if its just a string thats really helpful and have used it. Have refactored removing the `ProductsThresholdDefined ` type above for the moment. Thanks.